config_amboy.go

```diff
@@ -85,5 +93,21 @@ func (c *AmboyConfig) ValidateAndDefault() error {
 		c.LocalStorage = defaultAmboyLocalStorageSize
 	}
 
+	if c.GroupDefaultWorkers == 0 {
```

Comment:
Maybe validation for <1 or unreasonably large?

Reply:
I set the default if negative now. Since these are admin routes, I think maybe it's okay to allow large numberfs.